### ISSUE TITLE
Introduction of the Training Schedule

### ISSUE THREAD

**Issue by yanyc428** [RUCAIBox/JiuZhang3.0#2]
Hi~ Could you please give a detailed introduction of your JiuZhang3.0-8B training schedule during the pre-training stage and SFT stage? Thx~

**Comment by ToheartZhang** [comment]
We first pre-train the models (7B, 8B, and 8x7B) on our synthetic 4.6B math problem-solution pairs (CoT + Tool) and then fine-tune them on the multi-source instruction set (CoT + Tool). We adopt the Warmup-Stable-Decay learning rate scheduler with 3% warm-up ratio and 85% stable training ratio for 1 epoch in the whole training process.

Feel free to ask me if you have any remaining questions!
